### PR TITLE
Switched to using per item type glyphs for learning resources

### LINKED ISSUE #228: Bug: Icons in listing  page are directly using pureui.com instead of internal assets

### PR DESCRIPTION
**Pull request by carsongee**
Closes #228
[finished #97740486]
image
Defaults to using the little XML things if the type isn't known.

### CODE REVIEW

**Comment by pdpinch** [comment]
Looks a lot better now, although the icons themselves are pretty obscure.

File: ui/templates/repository.html [hunk 1]
```diff
@@ -24,116 +24,133 @@
...
+      {% endif %}
+      
+      {% if page.has_next %}
+      <a href="{% url 'repositories' repo.slug %}{{ qs_prefix }}page={{ page.next_page_number }}"><i
+                                                                                                     class="fa fa-angle-right"></i></a>
+      {% endif %}
+      {% endif %}
     </div>
+  </div>
+</div>
```
Comment (pdpinch) [inline ui/templates/repository.html:154]: is all this diff just because of tab changes?

Comment (carsongee) [inline ui/templates/repository.html:154]: Yeah, I ran indent-buffer in emacs, so the diff is huge

**Comment by noisecapella** [comment]
Looks pretty straightforward, I'll take a look at the page and then merge

**Comment by noisecapella** [comment]
Some of the icons look squashed, we may want to change them or their container to fit better, but that seems slightly out of scope

**Pull request merged**